### PR TITLE
Applied Routing to the project

### PR DESCRIPTION
**Pull request by AsimKhan2019**
Applied routing to the project and implemented Single Page Application, adding three pages to the project.

### CODE REVIEW

**Review by Ismailco — COMMENTED** [review]
Hi @AsimKhan2019, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is not using gitflow. kindly open a PR from the `spa` branch into the `dev` or `development branch, also `dev` branch should be the default branch

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by AsimKhan2019** [comment]
@Ismailco Thank you for the review, I have changed the pull request and the spa is now merging with the development branch. The development branch is the default branch in the project's settings.

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
+const App = () => (
+  <div className="App">
+    <Nav />
+    <Routes>
+      <Route exact path="/" element={<HomePage />} />
+      <Route path="/calculator" element={<CalculatorPage />} />
+      <Route path="/quote" element={<QuotePage />} />
+    </Routes>
+  </div>
+);
```
Comment (ShoiraTa) [inline src/App.js:16]: - [x] Everything is great, you followed SPA principles and app works as expected, only minor detail is remaining. Please make sure that the navbar is at the top of the page, currently it is sticked to body of the page and is dynamic:

Quotes | Calculator
------------- | -------------
Screenshot 2022-08-10 at 11 07 03 | Screenshot 2022-08-10 at 11 07 21



Wireframe: 

Screenshot 2022-08-10 at 11 09 07

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @AsimKhan2019 ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- You passed all the Linters! ✔️ 
- Professional pull request! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AsimKhan2019** [comment]
@ShoiraTa Thank you for the review, I will fix this right away

File: src/App.js (hunk 1 above)
Comment (AsimKhan2019) [reply src/App.js:16]: @ShoiraTa Thank you, I have removed the justify-content for the div from the CSS, and now the navigation is on top of the page.

**Comment by ShoiraTa** [comment]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @AsimKhan2019 , 

**Great work on making the changes 👏🏻** 
The `navbar` is on the top now 👍🏻 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AsimKhan2019** [comment]
@ShoiraTa Thank you for the approval

**Pull request merged**